lib/chrome.js

```
@@ -374,28 +325,18 @@
-
-                var pageList = HTTP.create("CURL")
-                    .setDataType("json")
-                    .open("GET", "http://127.0.0.1:" + this.debuggingPort + "/json")
-                    .send()
-                    .responseBody
-                ;
-
-                this.pageList = pageList;
-                return pageList;
+                var responseText = HTTP.get("http://127.0.0.1:" + this.debuggingPort + "/json");
+                var pageList = JSON.parse(responseText);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'pageList'. Consider to delete it or rename it.

'pageList' is defined here:


(lint/suspicious/noRedeclare)